{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"invalidSSN"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":1,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/dfc\/dfccollectssnerror\/dfccollectssnerrortype\/invalidssn"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.deserve.dfc\/documentation\/DFC\/DFCCollectSSNError\/DFCCollectSSNErrorType\/invalidSSN","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Failure while validating SSN."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"invalidSSN"}],"title":"DFCCollectSSNError.DFCCollectSSNErrorType.invalidSSN","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:3DFC18DFCCollectSSNErrorV0bC4TypeO10invalidSSNyA2EmF","modules":[{"name":"DFC"}]},"hierarchy":{"paths":[["doc:\/\/com.deserve.dfc\/documentation\/DFC","doc:\/\/com.deserve.dfc\/documentation\/DFC\/DFCCollectSSNError","doc:\/\/com.deserve.dfc\/documentation\/DFC\/DFCCollectSSNError\/DFCCollectSSNErrorType"]]},"references":{"doc://com.deserve.dfc/documentation/DFC/DFCCollectSSNError/DFCCollectSSNErrorType":{"role":"symbol","title":"DFCCollectSSNError.DFCCollectSSNErrorType","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"DFCCollectSSNErrorType"}],"abstract":[],"identifier":"doc:\/\/com.deserve.dfc\/documentation\/DFC\/DFCCollectSSNError\/DFCCollectSSNErrorType","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"DFCCollectSSNErrorType"}],"url":"\/documentation\/dfc\/dfccollectssnerror\/dfccollectssnerrortype"},"doc://com.deserve.dfc/documentation/DFC":{"role":"collection","title":"DFC","abstract":[],"identifier":"doc:\/\/com.deserve.dfc\/documentation\/DFC","kind":"symbol","type":"topic","url":"\/documentation\/dfc"},"doc://com.deserve.dfc/documentation/DFC/DFCCollectSSNError":{"role":"symbol","title":"DFCCollectSSNError","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"DFCCollectSSNError"}],"abstract":[{"type":"text","text":"This struct defines errors occurred during collection of SSN by SDK."}],"identifier":"doc:\/\/com.deserve.dfc\/documentation\/DFC\/DFCCollectSSNError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"DFCCollectSSNError"}],"url":"\/documentation\/dfc\/dfccollectssnerror"},"doc://com.deserve.dfc/documentation/DFC/DFCCollectSSNError/DFCCollectSSNErrorType/invalidSSN":{"role":"symbol","title":"DFCCollectSSNError.DFCCollectSSNErrorType.invalidSSN","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"invalidSSN"}],"abstract":[{"type":"text","text":"Failure while validating SSN."}],"identifier":"doc:\/\/com.deserve.dfc\/documentation\/DFC\/DFCCollectSSNError\/DFCCollectSSNErrorType\/invalidSSN","kind":"symbol","type":"topic","url":"\/documentation\/dfc\/dfccollectssnerror\/dfccollectssnerrortype\/invalidssn"}}}